packages/template-block/src/TemplateBlock.tsx

```diff
@@ -38,6 +38,12 @@ export const TemplateBlock = ({ appBridge }: BlockProps): ReactElement => {
 
     const handleNewPublication = async () => {
         if (selectedTemplate !== null) {
```

Comment:
Ah also, lets invert this for an early return @ryancarville 

```suggestion
        if (!selectedTemplate) {
        	return;
        }
```